collector/bcache_linux.go

```diff
@@ -134,14 +134,6 @@ func bcachePeriodStatsToMetric(ps *bcache.PeriodStats, labelValue string) []bcac
 			extraLabel:      label,
 			extraLabelValue: labelValue,
 		},
-		{
```

Comment:
We should make this conditional on the existance of `ps.CacheReadaheads` and only remove the stats when it's not available

Reply:
I just implemented that in a different, I only add the stats when the ps.CacheReadaheads exists and assume it isn't there by default. Just to save iterating and removing the element from the slice. Please let me know if that doesn't work so I can revisit it and fix it. Thank you!